### PR TITLE
Add Logo

### PR DESCRIPTION
**Pull request by olivcamj**

### CODE REVIEW

File: src/components/logo/logo.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-  // TODO: implement an actual logo
-  return <span className={`logo logo_size_${size}`}>Stub for the logo</span>;
+import './Logo.css';
+import logoImg from '../../images/icon.svg';
+import label from '../../images/iterative.svg';
+
+export function Logo() {
+  return (
+    <div className="logo">
+      <img className="logo logo__icon" src={logoImg} alt="Logo" />
```
Comment (julieg18) [inline src/components/logo/logo.js]: The alt name could be more accurate like "Iterative logo icon", and "Logo label saying Iterative".

File: src/components/logo/logo.css [hunk 2]
```diff
@@ -1,11 +1,90 @@
 .logo {
-  color: red;
+  display: flex;
+  align-items: center;
+  width: 114.26px;
```
Comment (julieg18) [inline src/components/logo/logo.css]: Nice job on sizing accuracy!

File: src/components/logo/logo.css [hunk 3]
```diff
@@ -1,11 +1,90 @@
...
+    margin: 3.2px 16px 3.2px 3.2px;
+  }
+
+  .logo__label {
+    width: 107.55px;
+    height: 20.25px;
+    margin: 5.87px 0;
+  }
+
+  .logo__text {
```
Comment (julieg18) [inline src/components/logo/logo.css]: logo__text styling is no longer needed

File: src/components/logo/logo.css [hunk 4]
```diff
@@ -1,11 +1,90 @@
...
+  margin: 2.4px 12px 2.4px 2.4px;
 }
 
-.logo_size_m {
-  font-size: 20px;
+.logo__label {
+  width: 80.66px;
+  height: 15.19px;
+  margin: 4.4px 0;
+  align-self: center;
```
Comment (julieg18) [inline src/components/logo/logo.css]: Your image doesn't complete cover your width/height area, causing you're text to be too small. Add a `object-fit: cover` to fix this.

File: src/images/icon.svg [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (julieg18) [inline src/images/icon.svg:1]: The icon you exported has some padding in it, causing the logo to be too small on the site. (You can make your width/height bigger or I can export a new svg that doesn't have the extra padding for you if needed 😄) 
image

**Review by julieg18 — CHANGES_REQUESTED** [review]
Great job so far! I love how much attention to detail you gave! If you have any questions about my comments, feel free to message me!

File: src/images/icon.svg (hunk 5 above)
Comment (olivcamj) [reply src/images/icon.svg]: Thank you for the correct image!

File: src/components/logo/Logo.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  width: 19.2px;
+  height: 19.2px;
+  margin: 2.4px 12px 2.4px 2.4px;
+}
+
+.logo__label {
+  width: 80.66px;
+  height: 15.19px;
+  margin: 4.4px 0;
+  align-self: center;
```
Comment (julieg18) [inline src/components/logo/Logo.css:18]: Your `.logo__label` still needs a `object-fit: cover` to be correctly sized.

**Review by julieg18 — CHANGES_REQUESTED** [review]
Nice job! After this last thing, I should be able to approve!

File: src/components/logo/Logo.css (hunk 6 above)
Comment (olivcamj) [reply src/components/logo/Logo.css:18]: Great catch! I missed that earlier.

**Review by julieg18 — APPROVED** [review]

**Pull request merged**